pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/parsers/MiroIdParsing.scala

```diff
@@ -0,0 +1,74 @@
+package uk.ac.wellcome.platform.transformer.sierra.transformers.parsers
+
+import org.apache.commons.lang3.StringUtils.leftPad
+
+import scala.util.matching.Regex
+
+object MiroIdParsing {
+  // Regex for parsing a Miro ID.  Examples of valid IDs, all taken from
+  // the Sierra data:
+  //
+  //    L0046161, V0033167F1, V0032544ECL
+  //
```

Comment:
Might be worth linking to some of the Slack discussions about what various parts of these IDs mean.

* Why do we only care about single-letter prefixes, when there are Miro prefixes with two letters (AS, FP)?
* Why seven digits?
* Why are there at most three suffix letters?

Reply:
I actually don't know the answers to these questions - this is existing code (and comment). Do you know any more about it?

Reply:
Good to distil those discussions into the comment for ease of use for the next dev to stumble on this.

Reply:
Agreed but as I say I don't have this information - could anyone point me towards it?